modules/n1ql/pages/n1ql-language-reference/with-recursive.adoc

```diff
@@ -212,8 +212,140 @@ SELECT * FROM rcte;
 ----
 ====
 
+.Examples handling hierarchical or tree-structured data
+====
+Before trying out these examples, you must:
+
+. Create a dataset containing employee information:
++
```

Comment:
Using numbered steps with code blocks in this example creates an inconsistency, because some some code blocks have titles and some don't, and it makes the code blocks look wonky. Use titled code blocks throughout the example.

```suggestion
.Create a dataset containing employee information
```